src/main/java/com/zaxxer/sansorm/SqlClosure.java

```diff
@@ -360,36 +330,6 @@ public static void quietClose(ResultSet resultSet)
       }
    }
 
-   /**
-    * Used to automatically close a Statement when the closure completes.
-    *
-    * @param statement the Statement to automatically close
-    * @return the Statement that will be closed (same as the input parameter)
-    */
-   @Deprecated
-   protected final <S extends Statement> S autoClose(S statement)
-   {
-      if (statement != null) {
-         closeStatements.add(statement);
-      }
-      return statement;
-   }
-
-   /**
-    * Used to automatically code a ResultSet when the closure completes.
-    *
-    * @param resultSet the ResultSet to automatically close
-    * @return the ResultSet that will be closed (same as the input parameter)
-    */
-   @Deprecated
-   protected final ResultSet autoClose(ResultSet resultSet)
-   {
-      if (resultSet != null) {
-         closeResultSets.add(resultSet);
```

Comment:
With the removal of ``autoClose(Statement)/autoClose(ResultSet)`` you can also remove the ``closeStatements`` and ``closeResultSets`` class members.

Reply:
@brettwooldridge, fixed all mentioned issues